backend/preprocess.py

```diff
@@ -39,6 +40,23 @@ def second_vote(mandates: List[Mandate]) -> List[Mandate]:
     return sort.second_vote(filtered_mandates)
 
 
+def second_vote_results(constituency: str, party: str) -> int:
+    if (second_vote_results := SECOND_VOTE_RESULTS.get(constituency)) != None:
+        return second_vote_results[party]
+    else:
+        return ""
+
+
+def constituency(constituency: str) -> str:
+    if "(" in constituency:
+        index = constituency.find("(")
+        constituency = constituency[0:index]
+        if "Landesliste" in constituency:
+            index = constituency.find("Landesliste")
+            constituency = constituency[index + 12 : -1]
```

Comment:
What's the `-1` saying? Until the end of the collection? If yes, you can also drop it.

Reply:
The constituency_name contains at the very end a space, which we don't want to include.

Reply:
I checked again and I was wrong, there are no spaces at the Landesliste. The space is actually in the statement before. Therefore I changed the Code (see below).